src/store/actions/actionCreators.js

```diff
@@ -0,0 +1,47 @@
+import { 
```

Comment:
please move functions that use fetch to separate file 'thunks.js'
since action creator won't go to the server for data or calculate anything. It's function that simply builds action from type and payload, nothing more